content/webapp/pages/exhibition-guide.tsx

```diff
@@ -177,6 +177,14 @@ function getTypeTitle(type: GuideType): string {
   }
 }
 
+function getTitle(stop: ExhibitionGuideComponent): string {
+  if (stop.title) {
```

Comment:
```suggestion
 return `${stop.number}. ${stop.title || stop.standaloneTitle}`
```

Maybe doesn't need a function in that case, just `${stop.number}. ${stop.title || stop.standaloneTitle}` inline? Not sure.

Reply:
nice, thanks